boards/imxrt1050-evkb/chip_layout.ld

```diff
@@ -0,0 +1,24 @@
+/* Memory layout for the i.MX RT 1050 EVKB
+ * rom = 2MB (LENGTH = 0x02000000)
+ * kernel = 256KB
+ * user = 256KB
+ * ram = 192KB */
+
+MEMORY
+{
+  hdr (rx)  : ORIGIN = 0x60000000, LENGTH = 0x2000
+  rom (rx)  : ORIGIN = 0x60002000, LENGTH = 0x3000000
+  prog (rx) : ORIGIN = 0x63002000, LENGTH = 0x1000000
```

Comment:
Does this work? I would expect a lot of MPU errors with large-ish apps because not many powers of two application sizes are aligned here. I think it would be better to make the `rom` 0x2000 smaller and start the `prog` at 0x63000000.

Reply:
So far, I had no issues with this setup, but I can see your point. I will update it.

Reply:
It is updated now, along with the GPIO interrupt support.